LanguageFeatures/Patterns/type_inference_A14_t03.dart

```diff
@@ -30,8 +28,4 @@ main() {
     dynamic e = 2.71;
     (v2) = e;
   });
-
-  final int Function(int) v3;
-  (v3) = foo;
-  v3.expectStaticType<Exactly<int Function(int)>>();
```

Comment:
This one should also give rise to a context type schema of `int Function(int)`, and that should enable generic function instantiation on `foo`. Again, this is "not a pattern thing".